protos/test_status.proto

```diff
@@ -0,0 +1,24 @@
+// Copyright 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wptdashboard;
+
+enum TestStatus {
+  TEST_STATUS_UNKNOWN = 0;
+  TEST_OK = 1;
+  TEST_ERROR = 2;
+  TEST_TIMEOUT = 3;
```

Comment:
Following the convention used in `Browser`, `Engine` and `OperatingSystem`: 

- `TEST_STATUS_UNKNOWN`
- `TEST_OK` => `OK`
- `TEST_ERROR` => `ERROR`
- `TEST_TIMEOUT` => `TIMEOUT`


Or alternatively: 

```go
enum TestStatus {
  UNKNOWN = 0;
  OK = 1;
  ERROR = 2;
  TIMEOUT = 3;
}
```

WDYT?

Reply:
Ignore the second part

Reply:
There's a name collision on `TIMEOUT` :cry:. That was the reason I aimed for a common-prefix strategy. WDYT?